packages/frontend/src/components/create-campaign/preview/index.tsx

```diff
@@ -371,21 +372,6 @@ export function CampaignPreview({
                     {t("newCampaign")}
                 </Button>
             </div>
-            <Confetti
-                numberOfPieces={600}
-                confettiSource={{
-                    x: 0,
-                    y: 0,
-                    w: width,
-                    h: height,
-                }}
-                run={true}
-                width={width}
```

Comment:
Are `height` and `width` still used somewhere? If not we can remove the `useWindowSize()` at line `51`.

Reply:
You're right, addressed in 52da68c0fe9b5c5d05cec17355867e19dafd9b0a